Paulstar200/Store-Manager
# #161832854 Consume Create products

**What does this PR do?**
Consumes the create product endpoint

**Description of Task to be completed?**
Consume the create product endpoint

**How should this be manually tested?**
Use the link [https://paulstar200.github.io/Store-Manager/UI/index.html](https://paulstar200.github.io/Store-Manager/UI/index.html)
OR
Clone my repository to your machine into a folder using git clone command as follows: `git clone https://github.com/Paulstar200/Store-Manager.git`
Go to the location to which you cloned the repository.
Change directory to Store Manager directory using `cd Store-Manager/UI`.
Switch to the appropriate branch by doing `git checkout ft-consume-create-product-161832854`
Launch the index.html file using a browser.
Log in using
`email:  vitalispaul48@live.com`
`password: manu2012`
Then click the create product link once open and create a product 
Ensure to input all the fields

**Any background context you want to provide?**
None

**What are the relevant pivotal tracker stories?**
#161832854
